app/validators/blocks/list_collector_content_validator.py

```diff
@@ -0,0 +1,22 @@
+from app.error_messages import FOR_LIST_NEVER_POPULATED
+from app.validators.blocks.block_validator import BlockValidator
+
+
+class ListCollectorContentValidator(BlockValidator):
```

Comment:
Might be an existing issue but is `validate_single_repeating_blocks_list_collector` on ListCollectorValidator correct?. It doesn't account for `ListCollectorContent` but either way the validation would need to happen if the initial block was content block hence needs to be done here?

Reply:
I thought we only needed that validation for list collectors? So we ensure only one list collector with repeating blocks exists for list=people or something but allow multiple for list collector content as its possible you'd want to do that

that said, I think this would block having a conditionally chosen list collector with repeating blocks for a list, so we may still have an issue

Reply:
Yes, I think you are right. In that case, all good.

> that said, I think this would block having a conditionally chosen list collector with repeating blocks for a list, so we may still have an issue

We'll revisit if it becomes an issue